### PR TITLE
Japanese translation has been completed.

### PR DESCRIPTION
**Pull request by kuragehimekurara1**
Please confirm that the Japanese translation has been completed.

### CODE REVIEW

**Comment by Chesire** [comment]
Thanks for this!
The translations are currently replacing the English strings, would it be possible to put them in `libraries/core/src/main/res/values-ja/strings.xml` and can look to get this merged in

**Comment by kuragehimekurara1** [comment]
> Thanks for this!
> The translations are currently replacing the English strings, would it be possible to put them in libraries/core/src/main/res/values-ja/strings.xml and can look to get this merged in

Should I create a new file and put strings.xml in it?

**Comment by Chesire** [comment]
If you create the required directories and put the new strings.xml there it should be all good.

The idea is to have the `values` folder which contains the strings file for English, and a `values-ja` folder which contains the strings file for Japanese

**Comment by kuragehimekurara1** [comment]
I understand!　　Do I just create a new pull request when I'm done?

**Comment by Chesire** [comment]
You can update this one (assuming the English strings are put back), or create a new one. Whichever you find easier for you 😃

**Comment by kuragehimekurara1** [comment]
Yes, sir.

**Comment by kuragehimekurara1** [comment]
I created a values-ja folder and added the strings.xml file translated into Japanese in it.  Also, the original strings.xml file has been restored to English.  How about this?

**Comment by kuragehimekurara1** [comment]
Also, if there is a part to be corrected, I would like to ask Chesire to correct it. Is it possible?

**Review by Chesire — COMMENTED** [review]
The changes look good, thanks for this.
One additional change to get this working is to update the `app/build.gradle` to include
```groovy
    defaultConfig {
        applicationId "com.chesire.nekome"
        minSdk 21
        targetSdk sdk_version
        versionCode 22040915 // Date of build formatted as 'yyMMddHH'
        versionName "1.5.3"
        testInstrumentationRunner "com.chesire.nekome.TestRunner"
        resConfigs "en", "ja"
    }
```
as the `resConfig` is only setup for `en` currently.

File: libraries/core/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+    <string name="login_header">Kitsuにログイン</string>
+    <string name="login_username">Kitsu eメール</string>
+    <string name="login_password">パスワード</string>
+    <string name="login_login">ログイン</string>
+    <string name="login_forgot_password">パスワードをお忘れですか？こちらから変更出来ます</string>
+    <string name="login_forgot_password_url">https://kitsu.io/password-reset</string>
+    <string name="login_sign_up">まだKitsuに登録されていませんか？ こちらからKitsuに登録出来ます</string>
+    <!-- This text should be exactly what part of login_sign_up should be a link -->
+    <string name="login_sign_up_link_target">サインアップ</string>
```
[the remark this answers is not in the document]
Comment (kuragehimekurara1) [reply libraries/core/src/main/res/values-ja/strings.xml:79]: How should I change the sign up part?

**Comment by kuragehimekurara1** [comment]
I got it

**Comment by kuragehimekurara1** [comment]
Updated build.gradle. Is this correct?

File: libraries/core/src/main/res/values-ja/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Chesire) [reply libraries/core/src/main/res/values-ja/strings.xml:79]: `login_sign_up` needs to contain the text from `login_sign_up_link_target` as it sets up a link using `login_sign_up_link_target`.
`login_sign_up_link_target` is not displayed to the user, its just used to lookup which part of the `login_sign_up` should contain a link.

It should be whichever part of `login_sign_up` is the equivalent to "Sign up"

**Comment by Chesire** [comment]
> Updated build.gradle. Is this correct?

Yup looks good 👍

File: libraries/core/src/main/res/values-ja/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (kuragehimekurara1) [reply libraries/core/src/main/res/values-ja/strings.xml]: Sorry for the late correction.
The translation has been corrected. How about this?

File: libraries/core/src/main/res/values-ja/strings.xml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+    <string name="login_header">Kitsuにログイン</string>
+    <string name="login_username">Kitsu eメール</string>
+    <string name="login_password">パスワード</string>
+    <string name="login_login">ログイン</string>
+    <string name="login_forgot_password">パスワードをお忘れですか？こちらから変更出来ます</string>
+    <string name="login_forgot_password_url">https://kitsu.io/password-reset</string>
+    <string name="login_sign_up">こちらからKitsuに登録出来ます</string>
+    <!-- This text should be exactly what part of login_sign_up should be a link -->
+    <string name="login_sign_up_link_target">こちらから</string>
```
Comment (Chesire) [inline libraries/core/src/main/res/values-ja/strings.xml:79]: I've made a change here to show what I think the string should be. It needs to have a matching section in the `login_sign_up` part. Its used just to set up some formatting for the string above.

File: libraries/core/src/main/res/values-ja/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Chesire) [reply libraries/core/src/main/res/values-ja/strings.xml]: Made a change to what I think it needs to be. If you are happy with the change I'll give it a test locally and look to merge it all in

[the remark this answers is not in the document]
Comment (kuragehimekurara1) [reply libraries/core/src/main/res/values-ja/strings.xml]: Thanks for changing the translation!
We're happy enough with it and you can merge it!

File: libraries/core/src/main/res/values-ja/strings.xml (hunk 2 above)
Comment (kuragehimekurara1) [reply libraries/core/src/main/res/values-ja/strings.xml:79]: Thanks for changing the translation!

**Comment by Chesire** [comment]
Thanks for this 👍

**Pull request merged**